pdp/__init__.py

```diff
@@ -15,6 +15,7 @@
 import yaml
 import static
 from beaker.middleware import SessionMiddleware
+from git import Repo
```

Comment:
Good points! My first attempt _did_ put the `get_commitish()` code into setup.py, however, this caused issues when trying to collect the values with `get_distribution()` as the added keyword (`revision`) is non-standard and did not get added to the package metadata. I'll take a look at the local version identifier and see if that will work instead.